piptools/exceptions.py

```diff
@@ -37,6 +37,8 @@ def __str__(self):
             lines.append(
                 "There are incompatible versions in the resolved dependencies."
             )
+            source_ireqs = getattr(self.ireq, "_source_ireqs", [])
+            lines.extend("- {}".format(ireq) for ireq in source_ireqs)
```

Comment:
IMO the `source_ireqs` should be sorted in case if there will be multiple packages it's more convenient to look at sorted list.. What do you think?

Reply:
Done